presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -139,13 +162,11 @@ public boolean isOutputTableWriterFragment()
         return outputTableWriterFragment;
     }
 
-    @JsonProperty
-    public StatsAndCosts getStatsAndCosts()
+    public Optional<StatsAndCosts> getStatsAndCosts()
```

Comment:
@rschlussel : I am wondering if `@JsonIgnore` can be the right pattern to represent such "unnecessary fields during  serialization" ?   See the discussion in https://github.com/prestodb/presto/pull/13507#discussion_r334253057 .